src/generated/resources/.cache/cache

```diff
@@ -0,0 +1,4 @@
+3dccc00bd1235b8950b30e4285a1a9db0ae17b12 assets/ancientology/lang/en_us.json
+87091ae6f7115ad2627b2ed09c69eca31ea4cb3e data/ancientology/loot_tables/blocks/test.json
+dd585fcc15f55cc62e8d4639ae2bb8be1e3b6c5f data/minecraft/tags/blocks/mineable/pickaxe.json
+dd585fcc15f55cc62e8d4639ae2bb8be1e3b6c5f data/minecraft/tags/blocks/needs_diamond_tool.json
```

Comment:
Iirc we don't need this file so it could be added to gitignore

Reply:
We do actually

Reply:
What's its purpose?